byway/src/main/webapp/style.css

```diff
@@ -7,3 +7,73 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#map{
+  height:500px; /* change to 40 vw */
+  width:500px; /* 5 vw padding*/
+}
+
+.route-container{
+  width: 1500px;
+  padding: 0 15px 0 15px;
+  margin: 20px auto 0 auto;
+  margin-right:auto;
+  margin-left:auto;
+  display:flex;
+}
+
+.box-outline {
+  border: solid 2px #fad3a7;
+  border-radius: 10px;
+  background-color: white;
+  height: 500px;
+  width: 350px;
+  margin: 0 0 0 30px;
+}
+
+.title{
+  border-radius: 10px;
+  height: 50px;
+  width: parent;
+}
+
+#box-outline-stops{
+  border: solid 2px #c2edda;
+}
+
+#schedule{
+  border: solid 2px #68d388;
+  background-color: #68d388;
+}
+
+#stops{
+  border: solid 2px #ffb766;
+  background-color: #ffb766;
+}
+
+h3{
```

Comment:
It might be good to make this more specific so it doesn't interfere if someone else used an h3 tag on another page

Reply:
done.